instrumentation/github.com/gin-gonic/gin/otelgin/test/gin_test.go

```diff
@@ -461,11 +461,25 @@ func TestMetrics(t *testing.T) {
 		name                        string
 		metricAttributeExtractor    func(*http.Request) []attribute.KeyValue
 		ginMetricAttributeExtractor func(*gin.Context) []attribute.KeyValue
+		requestTarget               string
+		expectHasRouteAttr          bool
+		expectStatus                int64
```

Comment:
We use `want` rather than `expected`.

```suggestion
		wantHasRouteAttr          bool
		wantStatus                int64
```